action.py

```diff
@@ -53,6 +53,34 @@ def _log(msg):
     print(msg, file=sys.stderr)
 
 
+def _download_ref_asset(ext):
+    repo = os.getenv('GITHUB_REPOSITORY')
+    ref = os.getenv("GITHUB_REF")
+
+    artifact = f"{os.getenv('GITHUB_REF_NAME')}-signed{ext}"
```

Comment:
Nitpick: This shouldn't cause issues as-is, but let's use a `NamedTemporaryFile` or similar for this.

Reply:
That, or just putting it under `/tmp` is probably sufficient 🙂

Reply:
good point! will resolve this when I swap in `requests`